queue.tf

```diff
@@ -69,3 +69,7 @@ output "processed_envelopes_queue_primary_listen_connection_string" {
 output "processed_envelopes_queue_primary_send_connection_string" {
   value = "${module.processed-envelopes-queue.primary_send_connection_string}"
 }
+
+output "envelopes_queue_max_delivery_count" {
```

Comment:
Do we need it in output ?

Reply:
Yes, otherwise you will not be able to retrieve this value for application (orchestrator to be specific) for our approach to manually dead letter messages

Reply:
don't we want to set something super hight (as disabling is apparently impossible) and just use the actual max in app settings?

Reply:
We had an increase to `288` - 24h window to try and process the message. It's reasonably high enough and if needed more - can be discussed in one of the refinements. Increasing to 300 ("nice round number") so it is more than safe to assume "we've tried for at least 24h". Still leaning towards `300 - 5` for orchestrator condition to dead letter